project_task_stock/tests/common.py

```diff
@@ -7,7 +7,7 @@
 from odoo.addons.base.tests.common import BaseCommon
 
 
-class TestProjectStockBase(BaseCommon):
+class TestProjectTaskStockBase(BaseCommon):
```

Comment:
Why renaming this here? It's not needed at all, and it messes a bit the commit diff.

Reply:
Because the unwritten rules (or at least mine) mean that it must be consistent with the module name, and otherwise it would be confusing with an existing module (`project_stock`) in core; anyway, it is already changed.

Reply:
Yeah, but now that it was not done in the previous migration PR, let's keep it for not having cross-changes in this migration PR.